<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionDialog</class>
 <widget class="QWidget" name="ConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Подключение к базе данных</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Адрес сервера</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditServerName">
     <property name="text">
      <string>etunbergelmir\SQLEXPRESS</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Имя базы данных</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditDatabaseName">
     <property name="text">
      <string>qtcourse</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Логин</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditUsername"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Пароль</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditPassword"/>
   </item>
   <item>
    <widget class="QPushButton" name="btnConnectToDatabase">
     <property name="text">
      <string>Подключиться</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
